presto-native-execution/presto_cpp/main/PrestoTask.cpp

```diff
@@ -590,10 +590,18 @@ void PrestoTask::updateExecutionInfoLocked(
   prestoTaskStats.outputPositions = 0;
   prestoTaskStats.outputDataSizeInBytes = 0;
 
-  prestoTaskStats.totalDrivers = veloxTaskStats.numTotalSplits;
-  prestoTaskStats.queuedDrivers = veloxTaskStats.numQueuedSplits;
-  prestoTaskStats.runningDrivers = veloxTaskStats.numRunningSplits;
-  prestoTaskStats.completedDrivers = veloxTaskStats.numFinishedSplits;
+  // TODO: The queuedDrivers count is likely inaccurate. Need to change to let
+  // Velox expose numQueuedDrivers and then come back to clean this up.
+  uint64_t numBlockedDrivers{0};
+  for (const auto& pair : veloxTaskStats.numBlockedDrivers) {
+    numBlockedDrivers += pair.second;
+  }
+  prestoTaskStats.queuedDrivers = veloxTaskStats.numTotalDrivers -
+      veloxTaskStats.numRunningDrivers - veloxTaskStats.numCompletedDrivers -
```

Comment:
Just wondering: any possibility you see this going negative?
 Agree with you, we should move the math in velox to get the queuedDrives instead of calculating here.

Reply:
I am not so sure about it. It might. But for now it's just a place holder. I'm going to have followups that is going to clean this part up soon.

Reply:
Maybe add `min(0, ...)` to safeguard it